nginx/Dockerfile

```diff
@@ -37,16 +38,14 @@ RUN ./configure \
 	--with-http_ssl_module \
 	--with-openssl=/tmp/openssl-${openssl_version} \
 	--with-openssl-opt="enable-weak-ssl-ciphers" \
-	--with-pcre
+	--without-http_rewrite_module
 RUN make -j$(nproc)
```

Comment:
The Nginx Makefile seems to compile OpenSSL by itself, so we don't actually need to compile OpenSSL here separately.